### ISSUE TITLE
Check if certain Image Directories are still needed or used

### ISSUE THREAD

**Issue by eSilverStrike** [Geeklog-Core/geeklog#887]
We should confirm if the image directories below are still needed or used for example

**public_html/images/_thumbs**

Added folders for Filemanager to store thumbnails. I can confirm this is being used still by the filemanager.

**public_html/images/library/File, Flash, Image and Media and all directories beneath** 

This seem to be a mixture of use according to GitHub comments. Seems to be used by filemanager as well. Is this needed since we already have the other directory above???

**/public_html/images/library/Image/icons/**
This seems to be used by the fckEditor? is it still needed? Our main editor now is the CKEditor. Here is the notes from the commit for the directory:

Addition of the fckEditor as an integrated option for the advanced_editor
Initial setup of updates which includes bulk of required new files and changes.
Version 2.0 Final Candidate of fckEditor
Includes an integrated version of the mcpuk image browser which has been modified to configure the image browser settings based on the GL config.
The mcpuk browser is an additional component but is a more advanced plugin
Image library base directory path defined in the main GL config.php but will be by default /images/library.

**Comment by mystralkk** [comment]
public_html/images/library/File, Flash, Image and Media and all directories beneath are used the Filemanager.  However, "/public_html/images/library/Image/icons/" directory is no longer used, I believe since the icon files in the directory are GIF files.

**Comment by eSilverStrike** [comment]
@mystralkk I have been checking out the image directories in the code. The _thumbs directory off of images is used for the File Manager. What are the Library directory used for by the File Manager? As far as I can tell they were added for the FCKeditor. If I look at the history of the directory they were added by Blaine in 2005 for the FCKeditor. Does the File Manager use them for a certain reason?

Here are the directories under images and what they are for (from my understanding):

/_thumbs/articles/ (used for article editor image thumbnails as well)
/_thumbs/library/
/_thumbs/userphotos/
- Folders for File Manager

/articles/
- Folder for Article Images

/buttons/
- Folder for image buttons/badges (php, geeklog, mysql, valid-css, valid-html)

/cache/
- Cache for Article Image Uploads

/library/File/
/library/Flash/
/library/Image/
/library/Media/
- Seems to be for FCKeditor .... and for file manger?

/topics/
- For Topic images

/userphotos/
- For User Photos

**Comment by eSilverStrike** [comment]
@mystralkk Okay after checking again I think I got it all figured out. I updated the above comment with the latest. The library directories are used by CKeditor (see #890) and File Manager. The only one I don't see used is Media but maybe I missed the toolbar button it uses or something....

So I plan on removing the following directories and updated the checks for them in the installer, enviroment check, and clear cache:
/cache/
/_thumbs/library/
/buttons/ (contains old button images from 2005 that no one uses anymore)

Sound good?
